src/extensions/scratch3_boost/index.js

```diff
@@ -1944,7 +1944,7 @@ class Scratch3BoostBlocks {
      * @private
      */
     _isColor (args) {
-        switch (args) {
```

Comment:
It was switch on an array of arguments before?

Reply:
yup, that was also breaking the "any" condition I think, which I didn't notice